src/test/java/io/vertx/core/ComplexHATest.java

```diff
@@ -40,11 +37,11 @@
  */
 public class ComplexHATest extends VertxTestBase {
 
-  protected ClusterManager getClusterManager() {
+  @Override protected ClusterManager getClusterManager() {
```

Comment:
Please configure the formatter of your IDE or text editor. The annotation should be on a separate line above.